tutorials/cyber/raw_data_generator/README.md

```diff
@@ -0,0 +1,43 @@
+# LANL WEL Raw Data Generator
```

Comment:
Would it make sense to include an explanation of the "WEL" acronym, and what these types of logs represent?

Maybe also include links to the specific event code templates this notebook deals with.

Reply:
Sure, I have just updated and spelled out the acronym and also included explanation of a specific event code template and corresponding link